packages/docs/src/pages/en/getting-started/frequently-asked-questions.md

```diff
@@ -162,7 +162,7 @@ export default {
 
 - **My _/_ link is active when I'm on _/home_ page.**{ #link-active-home }
 
-  Add the **exact** to the link that points to absolute /. For more information on this, you can visit the official Vue router [documentation](https://router.vuejs.org/en/api/router-link.html).
+  Add the **exact** to the link that points to absolute /. For more information on this, you can visit the official Vue router [documentation](https://router.vuejs.org/api/#routerlink).
```

Comment:
`exact` has been removed <https://router.vuejs.org/guide/migration/index.html#removal-of-the-exact-prop-in-router-link>

Also <https://router.vuejs.org/api/interfaces/RouterLinkProps.html> page has some more information than the page you've proposed

Reply:
Okay, so what you are suggesting is to remove that line of text completely then? (Considering that information is outdated now)

Reply:
Okay, what are you suggesting to change then? Changing the link to the second one you sent?

Reply:
TBH not sure, since my link doesn't really provide more information on `exact` this might be confusing to users. But probably it's still better than pointing to non existing page